### PR TITLE
TypeORM

### PR DESCRIPTION
**Pull request by SaltyQuetzals**
`RoomController.playMove` is quite slow (~600ms) because of inefficient implementations. I don't know SQL well enough to profile the slowdown, though.

### CODE REVIEW

File: server/index.ts [hunk 1]
```diff
@@ -1,122 +1,115 @@
...
+      } = await roomController.playMove(socket.id, cards);
+      io.to(roomCode).emit(SocketEvent.MOVE_PLAYED, {
+        name,
+        cards: playedCards,
+        updated,
+        board,
       });
-    } catch (err) {}
+    } catch (err) {
+      socket.emit('eror', err.toString());
```
Comment (jaflo) [inline server/index.ts:88]: Should probably be error for consistency

File: server/index.ts [hunk 2]
```diff
@@ -1,122 +1,115 @@
...
-        io
-          .to(affectedRoom.roomCode)
-          .emit(SocketEvent.PLAYER_DISCONNECTED, socket.id)
+      io.to(affectedRoom.roomCode).emit(
+        SocketEvent.PLAYER_DISCONNECTED,
+        socket.id
       );
     } catch (err) {
-      socket.emit("error", err.toString());
+      socket.emit('err', err.toString());
```
Comment (jaflo) [inline server/index.ts:103]: Also "error". Note for future: should we add what the error was in response to? Does not need to be done now.

File: server/index.ts (hunk 1 above)
Comment (SaltyQuetzals) [reply server/index.ts:88]: Good catch. This was a typo.

File: server/index.ts (hunk 2 above)
Comment (SaltyQuetzals) [reply server/index.ts]: Good catch!

**Pull request merged**